.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+  "trailingComma": "es5",
+  "endOfLine": "auto",
+  "overrides": [
+    {
+      "files": ["*.md"],
+      "options": {
+        "proseWrap": "preserve"
+      }
+    },
+    {
+      "files": ["*.{yml,yaml}"],
+      "options": {
+        "singleQuote": true
+      }
+    }
```

Comment:
Setting `singleQuote` to true makes more changes than leaving it at its default value, so I'd argue this override should go away